im-sdk-cli/src/main/java/com/easemob/im/cli/cmd/CreateCmd.java

```diff
@@ -110,4 +132,31 @@ public void group(@Option(names = "--private", description = "create a private g
                     .block();
         }
     }
+
+    static class MemberArgGroup {
+        @Option(names = "--group", description = "add a group member")
+        boolean group;
+
+        @Option(names = "--room", description = "add a room member")
+        boolean room;
+    }
+
+    @Command(name = "member", description = "Add user to group or room.")
+    public void member(@Parameters(index = "0", description = "the group or room's id") String id,
```

Comment:
Let's use --to-group={groupId} or --to-room={roomId} here.
And, remove the index="1" from username Parameter.